.travis.yml

```diff
@@ -1,4 +1,9 @@
-language: c
-compiler: gcc
```

Comment:
Unfortunately Travis doesn't seem to support different `language`s. So you can either specify multiple compliers or multiple Python versions.

I wanted to test with both Python 2.7 and Python ≥ 3.3, but now as you committed 30e10d914ff10910 to specify multiple compilers, I reverted my change and it tests only Python 2.7 now.

If in future Travis moves to a newer OS that has Python ≥ 3.3 (Ubuntu 12.04 doesn't), we will be able to install it using `apt-get` and test with it.